setup.py

```diff
@@ -0,0 +1,5 @@
+# Local pre-commit hook setup
+from __future__ import annotations
+
+from setuptools import setup
+setup()
```

Comment:
Is there a way to rename setup.py and setup.cfg to some other filename? Otherwise, they look like a setup for the `sonic-mgmt` repo.

Reply:
I can only use 'setup.py' or 'pyproject.toml', so I think maybe I can add some comments at the top of `setup.py` and `setup.cfg`

Reply:
Use `pyproject.toml` instead of `setup.py`